app/controllers/search_controller.rb

```diff
@@ -5,6 +5,7 @@ def index
     @form = SearchForm.new(search_form_params)
     if @form.valid?
       @result = FirmRepository.new.search(@form.to_query, page: page)
+      redirect_to "/404.html" if @result.total_pages < page
```

Comment:
@shaley91 could you use single-quotes in this instance please?

"/404.html" --> '/404.html'